src/pages/UserHome.jsx

```diff
@@ -109,35 +107,40 @@ export default function UserHome() {
             Add Scholarship
           </Button>
         </Grid>
+        {scholarships.length === 0 && (
+          <Grid container className={classes.bannerRoot}>
+            <Grid item xs={3}>
+              <InboxIcon style={{ fontsize: 600 }} />
```

Comment:
The [Icon property](https://material-ui.com/api/icon/#props) is called `fontSize` not `fontsize`.